recipes/prody/meta.yaml

```diff
@@ -53,18 +53,20 @@ about:
   license: MIT
   license_family: MIT
   license_file: LICENSE.rst
-  summary: 'ProDy is a free and open-source Python package for protein structure, dynamics, and sequence analysis.'
+  summary: 'ProDy is a free and open-source Python package for protein structure, dynamics, and sequence analysis, developed by the
+    Bahar lab.'
   description: |
     ProDy is a free and open-source Python package for protein 
-    structure, dynamics, and sequence analysis. It allows for 
-    comparative analysis and modeling of protein structural 
-    dynamics and sequence co-evolution. Fast and flexible ProDy 
-    API is for interactive usage as well as application development. 
-    ProDy also comes with several analysis applications and a 
-    graphical user interface for visual analysis.
+    structure, dynamics, and sequence analysis, developed by the
+    Bahar lab. It allows for comparative analysis and modeling of 
+    protein structural  dynamics and sequence co-evolution. Fast 
+    and flexible ProDy API is for interactive usage as well as 
+    application development. ProDy also comes with several analysis 
+    applications and a graphical user interface for visual analysis.
   doc_url: http://prody.csb.pitt.edu/manual/
   dev_url: https://github.com/prody/ProDy
 
 extra:
   recipe-maintainers:
+    - jamesmkrieger
```

Comment:
@jamesmkrieger, conda-forge requires prospective maintainers to post a comment to confirm they are willing to be listed, could you oblige?

If any of your colleagues want to be added, it would be great if they could do the same.

Reply:
Thanks Simon,

I confirm that I am willing to be listed. I've asked some of my colleagues too.

Best wishes
James